src/dotnet/ToolManifest/LocalizableStrings.resx

```diff
@@ -138,4 +141,7 @@
   <data name="VersionIsInvalid" xml:space="preserve">
     <value>version {0} is invalid.</value>
   </data>
+  <data name="ToolMissingVersion" xml:space="preserve">
+    <value>field version is missing.</value>
```

Comment:
Should this start with Field? Capital F?

Reply:
fixed